web-app/js/ext-3.3.1/ext-all-debug.js

```diff
@@ -40651,19 +40651,11 @@ Ext.form.DateField = Ext.extend(Ext.form.TriggerField,  {
     
     showToday : true,
     
-    
     startDay : 0,
-    
-    
-    
-    
-    
-    
 
-    
-    defaultAutoCreate : {tag: "input", type: "text", size: "10", autocomplete: "off"},
+    compareText : "From date {0} should be before To date {1}",
 
-    
+    defaultAutoCreate : {tag: "input", type: "text", size: "10", autocomplete: "off"},
```

Comment:
**You cant change any Extjs (core) file!!!!!!!!!**

Notice that  production includes
`<script src="/js/ext-3.3.1/ext-all.js" type="text/javascript"></script>`
Whereas while developing we are using
`<script type="text/javascript" src="/js/ext-3.3.1/ext-all-debug.js">`

The solution may be to use `js/portal/prototypes/Ext.js` if essential?
(It seem this files purpose has morphed anyhow)